src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/**
+* Represents the static data of a user.
+*/
+public class User {
+    private final String userID;
+    private final String username;
+    private final String firstName;
+    private final String lastName;
+    private final String profilePicture;
+    private final ArrayList<String> gamesCreated;
```

Comment:
For "gamesCreated", I expected to have a List<Game>. Since we have List<String>, I think it might be more clear to have List<String> gameIDsCreated.

Reply:
Good point. I didn't know the difference between a List and an ArrayList, so I always defaulted to ArrayList because it was the one I had more experience with. I'll change all of these to List because it seems to be more versatile.

Also I don't think putting "ID" in the name of the variable is necessary because the default assumption over all the code is that when referencing a game, stage, hint, or user, we always use ID. Also this would require changing the names of a lot more variables, which would take a long time.